### PR TITLE
[CBRD-24419] add new testcase for query specification of the boot_define_view_ function change.

### PR DESCRIPTION
**Pull request by tw-kang**
http://jira.cubrid.org/browse/CBRD-24419

### CODE REVIEW

**Review by ssihil — APPROVED** [review]

**Review by kwonhoil — APPROVED** [review]

File: sql/_34_fig/cbrd_24419/cases/cbrd_24419_1.sql [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+ *   - db_index
+ *   - db_index_key
+ *   - db_auth
+ *   - db_trig
+ *   - db_partition
+ *   - db_stored_procedure
+ *   - db_stored_procedure_args
+ *   - db_server
+ *   - db_synonym
+ *   - skip checking db_charset, db_collation doesnt have owner_name attribute.
```
Comment (youngjinj) [inline sql/_34_fig/cbrd_24419/cases/cbrd_24419_1.sql]: db_charset and db_collation in the system views also changed the query specification.
So we have to check whether the result is the same by testing.

It has nothing to do with the owner_name.

File: sql/_34_fig/cbrd_24419/cases/cbrd_24419_1.sql [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+create table t3 as subclass of t3_s (c1 int, c3 t3_d, c4 sequence of t3_d) comment 'u0 {dba} > t3 (inherit)'
+  method class m0(sequence of t3_d) t3_d function f0, m1(varchar(255)) int function f1
+  file '$home/method'
+  inherit class m0 of t3_s as m0_s, m1 of t3_s as m1_s, c1 of t3_s as c1_s;
+create table t4 (c1 int primary key, c2 int, c3 varchar(255)) comment 'u0 {dba} > t4';
+create index i1 on t4 (c1, c2) comment 'u0 {dba} > t4 > i1 (index)' invisible;
+create index i2 on t4 (c2) where c2 = 0 comment 'u0 {dba} > t4 > i2 (filter)';
+create index i3 on t4 (replace (c3, ' ', '')) comment 'u0 {dba} > t4 > i3 (function)';
+create table t5 (c1 int unique, c2 int) comment 'u0 {dba} > t5';
+create table t5_c (c1 int unique, c2 int) comment 'u0 {dba} > t5_c (copy )';
```
Comment (youngjinj) [inline sql/_34_fig/cbrd_24419/cases/cbrd_24419_1.sql]: It is trivial.
```suggestion
create table t5_c (c1 int unique, c2 int) comment 'u0 {dba} > t5_c (copy)';
```

**Comment by youngjinj** [comment]
This test should be performed after all user tables have been created.
Test user u0, don't drop all tables for user u0.

The purpose of this test is to check that other users' tables are visible, which should not be visible.

**Comment by tw-kang** [comment]
> This test should be performed after all user tables have been created. Test user u0, don't drop all tables for user u0.
> 
> The purpose of this test is to check that other users' tables are visible, which should not be visible.

This problem could be covered in cbrd_24419_2.sql.
When selecting u2 it doesn't show other groups or other user's tables.

**Comment by youngjinj** [comment]
u0 is a member user of the dba group.
Unable to validate that a member user of the dba group can view tables of other groups or member users of other groups.

And u4 granted select permission only to u5.
Unable to validate that this table is visible in system views by users u1, u2, u3.

**Review by kwonhoil — APPROVED** [review]

File: sql/_34_fig/cbrd_24419/answers/cbrd_24419.answer [hunk 3]
```diff
@@ -0,0 +1,4259 @@
...
+    
+null     
+
+===================================================
+    
+null     
+
+===================================================
+class_name    owner_name    class_type    is_system_class    tde_algorithm    partitioned    is_reuse_oid_class    collation    comment    
+t1     DBA     CLASS     NO     NONE     NO     NO     iso88591_bin     u0 {dba} > t1
```
Comment (youngjinj) [inline sql/_34_fig/cbrd_24419/answers/cbrd_24419.answer]: Is it the result of executing the cbrd_24419.sql file?
In the cbrd_24419.sql file, the DBA does not create any tables.
Different results seem to be mixed.
Please check.

**Review by ssihil — APPROVED** [review]

**Review by youngjinj — APPROVED** [review]

**Pull request merged**